Makefile

```diff
@@ -103,24 +103,19 @@ install:
 	cp -Ra dbicdh "$(DESTDIR)"/usr/share/openqa/dbicdh
 
 
-.PHONY: checkstyle
-checkstyle: test-shellcheck test-yaml
-	PERL5LIB=lib/perlcritic:$$PERL5LIB perlcritic lib
-
 .PHONY: test
 ifeq ($(TRAVIS),true)
 test: run-tests-within-container
 else
 ifeq ($(CHECKSTYLE),0)
```

Comment:
@Martchus if I remember correctly you thought that needing two flags to unselect style checks and selecting individual tests could be improved. E.g. currently we need:

```sh
make test CHECKSTYLE=0 PROVE_ARGS=t/42-my-test.t
```

to run just a single test. Of course one can just use

```sh
make test-with-database PROVE_ARGS=t/42-my-test.t
```

instead but it's not less typing :)

We could do something like

```sh
make test TESTS=t/42-my-test.t
```

which internally switches off CHECKSTYLE. Would this be good? Any further ideas?

Reply:
It's less typing for those who use tab completion ;-)

Reply:
I like `TESTS` in the sense that `PROVE_ARGS` takes more arguments than just the name of the test. You use it specify the *harness* for instance.

Reply:
Covered this in https://github.com/os-autoinst/openQA/pull/2847 . You can now approve this PR here :)